courses/machine_learning/deepdive2/introduction_to_tensorflow/solutions/load_images_tf.data.ipynb

```diff
@@ -21,7 +21,7 @@
     "In this notebook, we load an image dataset using tf.data.  The dataset used in this example is distributed as directories of images, with one class of image per directory.\n",
```

Comment:
<p>Well done - using 1a and 1b!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1090/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>